st2exporter/st2exporter/exporter/dumper.py

```diff
@@ -126,3 +135,33 @@ def _get_file_name(self):
         file_name = self._file_prefix + timestring + '.' + self._file_format
         file_name = os.path.join(self._export_dir, file_name)
         return file_name
+
+    def _update_marker(self, batch):
```

Comment:
When exporter is running on different boxes, each box needs its own marker since they all can go at different speeds. So no locking is required. Anyways, I am intentionally punting the distributed aspects of exporter for now to keep the prototype simple.